Dockerfile

```diff
@@ -1,184 +1,174 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04 AS compiler-common
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
 
-# Based on
-# https://switch2osm.org/manually-building-a-tile-server-18-04-lts/
+RUN apt-get update \
+&& apt-get install -y --no-install-recommends \
+ ca-certificates gnupg lsb-release locales \
+ wget curl \
+ git-core unzip unrar \
+&& locale-gen $LANG && update-locale LANG=$LANG \
+&& sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+&& wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+&& apt-get update && apt-get -y upgrade
+
+###########################################################################################################
+
+FROM compiler-common AS compiler-stylesheet
+RUN cd ~ \
+&& git clone --single-branch --branch v5.4.0 https://github.com/gravitystorm/openstreetmap-carto.git --depth 1 \
+&& cd openstreetmap-carto \
+&& sed -i 's/, "unifont Medium", "Unifont Upper Medium"//g' style/fonts.mss \
+&& sed -i 's/"Noto Sans Tibetan Regular",//g' style/fonts.mss \
+&& sed -i 's/"Noto Sans Tibetan Bold",//g' style/fonts.mss \
+&& sed -i 's/Noto Sans Syriac Eastern Regular/Noto Sans Syriac Regular/g' style/fonts.mss \
+&& rm -rf .git
+
+###########################################################################################################
+
+FROM compiler-common AS compiler-helper-script
+RUN mkdir -p /home/renderer/src \
+&& cd /home/renderer/src \
+&& git clone https://github.com/zverik/regional \
+&& cd regional \
+&& rm -rf .git \
+&& chmod u+x /home/renderer/src/regional/trim_osc.py
 
-# Set up environment
-ENV TZ=UTC
+###########################################################################################################
+
+FROM compiler-common AS final
+
+# Based on
+# https://switch2osm.org/serving-tiles/manually-building-a-tile-server-18-04-lts/
+ENV DEBIAN_FRONTEND=noninteractive
 ENV AUTOVACUUM=on
 ENV UPDATES=disabled
+ENV REPLICATION_URL=https://planet.openstreetmap.org/replication/hour/
```

Comment:
Das war bei uns afaik eine Konfigurationsoption. Kannst Du prüfen, ob das irgendwie interagiert?

Reply:
Die URL stand bei uns explizit in `openstreetmap-tiles-update-expire.sh`, aber die Stelle werde ich komplett von Upstream übernehmen.